modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -0,0 +1,183 @@
+// Module included in the following assemblies:
+// * observability/network_observability/netobserv-cli-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-netobserv-cli-commands_{context}"]
+= Network Observability CLI reference 
+The Network Observability CLI (`oc netobserv`) is a CLI tool for capturing flow data and packet data for further analysis.
+
+.`oc netobserv` syntax
+[source,terminal]
+----
+$ oc netobserv [<command>] [<feature_command>] [<subcommands>] <1>
+----
+1. Feature commands can only be used with the `oc netobserv flows`. They cannot be used with `oc netobserv packets`.
+
+[cols="3a,8a",options="header"]
+.Basic commands
+|===
+|Option| Description
+
+| `flows`
+| Capture flows information. For subcommands, see the "Flows subcommands" table on this page. 
```

Comment:
The table is named slightly different than this. Also, for the packet line, you use the section not the title. So I'd pick one and be consistent between the two.

Also, I think you can also remove "on this page"

```suggestion
| Capture flows information. For subcommands, see the "Flow capture subcommands" section. 
```